.release-notes/4649.md

```diff
@@ -0,0 +1,13 @@
+## Add ability to trace runtime events in chromium json format #4649
+
+We have added the ability to trace runtime events for pony applications. Runtime tracing is helpful in understanding how the runtime works and for debugging issues with the runtime.
+
+The tracing can either write to file in the background or work in a "flight recorder" mode where events are stored into in memory circular buffers and written to stderr in case of abnormal program behavior (SIGILL, SIGSEGV, SIGBUS, etc).
+
+Chromium json format is documented at:
```

Comment:
Can you introduce that the format for events is in this format before noting where it is documented?

Also can you use markdown links instead of this plain text style?

I think a paragraph sans links about the format and viewer and then a markdown listing of relevant links would be good.

Reply:
updated